compiler/src/iree/compiler/Dialect/LinalgExt/IR/LinalgExtOps.cpp

```diff
@@ -1856,6 +1856,8 @@ LogicalResult CustomOp::verify() {
   return success();
 }
 
+/// Start `LinalgFusionInterface` implementation.
```

Comment:
I think we can consider to implement these methods in `LinalgFusionInterfaceImpl.cpp`. Perhaps @IanWood1 can help on this in a follow-up??

Reply:
I am not sure we want to split op definitions this way. Ill stick with the current split, but I am not convinced it should be this way.